docs/fundamentals/connection/tls.txt

```diff
@@ -0,0 +1,192 @@
+.. _laravel-tls:
+
+========================
+Enable and Configure TLS
+========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, security, connection options
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the TLS protocol to secure 
+your connection to a MongoDB deployment. To configure your connection to
+use TLS, enable the TLS option and provide your certificates for
+validation when configuring a connection in your application's
+``config/database.php`` file.
```

Comment:
Suggestion:

I think the "when" could be omitted since it doesn't necessarily need to be a specific point in time.

```suggestion
validation in your application's
``config/database.php`` file.
```